RecoTauTag/RecoTau/plugins/RecoTauBuilderConePlugin.cc

```diff
@@ -382,6 +456,12 @@ RecoTauBuilderConePlugin::return_type RecoTauBuilderConePlugin::operator()(
   if ( primaryVertexRef.isNonnull() )
     tauPtr->setVertex(primaryVertexRef->position());
 
+  // Set missing tau quantities
+  setTauQuantities(*tauPtr,
```

Comment:
(Just to understand the logic): here you modify the *tauPtr which was got a few lines above. And this is what you push back in output. Correct?